packages/blocks/README.md

```diff
@@ -720,6 +756,30 @@ _Returns_
 
 Registers a new block variation for the given block type.
```

Comment:
We can reference here the documentation page:

https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-variations.md